docs/src/content/hardhat-runner/docs/guides/verifying.md

```diff
@@ -4,23 +4,46 @@ Once your contract is ready, the next step is to deploy it to a live network and
 
 Verifying a contract means making its source code public, along with the compiler settings you used, which allows anyone to compile it and compare the generated bytecode with the one that is deployed on-chain. Doing this is extremely important in an open platform like Ethereum.
 
-In this guide we'll explain how to do this in the [Etherscan](https://etherscan.io/) explorer, but there are other ways to verify a contract, for example with [Sourcify](https://sourcify.dev/).
+In this guide we'll explain how to do this in the [Etherscan](https://etherscan.io/) explorer.
```

Comment:
Maybe we should add some note about the other plugin, somewhere in this guide. Something like: If your contracts weren't deployed with Hardhat Ignition, take a look at this plugin instead.

Reply:
We have a follow on tooltip:

```
:::tip

If you wish to verify a contract deployed outside of Hardhat Ignition or if you'd like to verify on Sourcify instead of Etherscan, you can use the [hardhat-verify plugin](/hardhat-runner/plugins/nomicfoundation-hardhat-verify).
```